spec/person_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './person'
+
+describe Person do
+  before :each do
+    @person = Person.new 20, 'Lucas'
+  end
```

Comment:
- [ ] Let ensure that there are unit tests for all the methods in all the classes, please kindly write unit test for the `can_use_services?`, `correct_name`, `add_rental`  methods in the Person class.